.github/workflows/release.yml

```diff
@@ -17,34 +17,69 @@ jobs:
           GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
         run: |-
           echo 'GITHUB_TOKEN=${{secrets.GITHUB_TOKEN}}' > .release-env
-      - name: draft release
+      - name: build and upload goreleaser binaries
         run: make release
+
+  rocksdb-x86_64:
+    strategy:
+      matrix:
+        os: [ubuntu-latest, macos-latest]
+    needs: ["goreleaser"]
+    runs-on: ${{ matrix.os }}
+    environment: release
+    steps:
+      - uses: cachix/install-nix-action@v16
+      - uses: cachix/cachix-action@v10
+        with:
+          name: cronos
+          signingKey: "${{ secrets.CACHIX_SIGNING_KEY }}"
+      - name: build linux/mac x86_64 rocksdb binaries
+        run: |
+          PLATFORM="$(uname -s)_x86_64"
+          FLAKE="github:${GITHUB_REPOSITORY}/${GITHUB_REF_NAME}"
+          nix build ${FLAKE}#cronosd-tarball
+          cp result cronos_${GITHUB_REF_NAME}-rocksdb_${PLATFORM}.tar.gz
+          nix build ${FLAKE}#cronosd-testnet-tarball
+          cp result cronos_${GITHUB_REF_NAME}-testnet-rocksdb_${PLATFORM}.tar.gz
+          echo 'FILES<<EOF' >> $GITHUB_ENV
+          ls -1 *.tar.gz >> $GITHUB_ENV
+          echo 'EOF' >> $GITHUB_ENV
+          cat $GITHUB_ENV
+      - name: upload linux/mac x86_64 rocksdb binaries
+        uses: softprops/action-gh-release@v1
+        with:
+          draft: true
+          files: "${{ env.FILES }}"
+
+  rocksdb-aarch64-linux:
+    needs: ["goreleaser"]
+    runs-on: ubuntu-latest
+    environment: release
+    steps:
       - uses: cachix/install-nix-action@v16
+      - run: |
+         DEBIAN_FRONTEND=noninteractive
+         sudo apt-get update -q -y && sudo apt-get install -q -y qemu-system-aarch64 qemu-efi binfmt-support qemu-user-static
+         mkdir -p ~/.config/nix
+         sudo bash -c "echo system-features = aarch64-linux arm-linux >> /etc/nix/nix.conf"
```

Comment:
didn't fully understand how, but it seems to work.
it use some binfmt trick on linux to build arm64 binary on x86_64 machine without doing complete cross-compiling.